pm_icecon/bt/compute_bt_ic.py

```diff
@@ -724,24 +716,32 @@ def coastal_fix(
     is_considered = is_temp0 & is_rolled_land
 
     # args to np.roll are opposite of fortran index offsets
+    # TODO: one less roll operation here than in `_conc_change124`. Should there
+    # be?
```

Comment:
Here in the logic for the third conc2 change, we  have one less roll operation and one less conditional than in `_conc_change124`. Is this a mistake, or intentional?